Contribute/dotnet/dotnet-contribute.md

```diff
@@ -120,7 +139,7 @@ Each complete sample that you create should contain a *readme.md* file. This fil
 
 Your topic will also contain links to the sample. Link directly to the sample's folder on GitHub.
 
-### Writing a new snippet or sample
+### Writing a new sample
```

Comment:
Can we link to a representative sample that illustrates what we're saying in this section?